lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
> Because of that, we should avoid adding things to `getReport*` that are specific to the main report and never used elsewhere

arg, see this is why we need a diagram :S

If none of the `getReportJS` scripts end up used in viewer, how does that make the browserified bundle bigger?